tests/formats_test/luks_test.py

```diff
@@ -49,6 +49,10 @@ def test_resize(self):
         self.fmt.update_size_info()
         self.assertEqual(self.fmt.current_size, new_size)
 
+    def tearDown(self):
+        self.fmt.teardown()
+        super(LUKSTestCase, self).tearDown()
```

Comment:
This should ideally be added with ``add_cleanup()``, but that's a general issue of these tests so I'm okay with being consistent here.